app/views/unattended/provisioning_templates/snippet/kickstart_kernel_options.erb

```diff
@@ -32,7 +32,7 @@ snippet: true
   options.push("kssendmac", "ks.sendmac", "inst.ks.sendmac")
 
   # networking credentials
-  raise("Dual-stack provisioning not supported") if subnet4 && subnet6
```

Comment:
This and the other change to this file do not seem like they are related to this issue and probably deserve their own separate PR

Reply:
Ah this is a leftover from https://github.com/theforeman/foreman/pull/8447 that should not be here for sure.

Reply:
can you remove it then? :)